plugins/volumeFilter_test.go

```diff
@@ -229,13 +298,31 @@ func TestMakeFilterVolume(t *testing.T) {
 	}
 }
 
-func TestVolumeFilterFn(t *testing.T) {
-	db := connectTestDb()
-	dailyVolumeByDateQuery, e := makeDailyVolumeByDateQuery(db, "", "native", "native", []string{}, []string{})
-	if e != nil {
-		t.Log("could not make empty config volume query")
-		return
+func makeManageSellOffer(price, amount string) *txnbuild.ManageSellOffer {
+	if amount == "" {
+		return nil
+	}
+
+	return &txnbuild.ManageSellOffer{
+		Buying:  txnbuild.NativeAsset{},
+		Selling: txnbuild.NativeAsset{},
+		Price:   price,
+		Amount:  amount,
+	}
+}
+
+func makeTestVolumeFilterConfig(baseCap, quoteCap float64) *VolumeFilterConfig {
+	return &VolumeFilterConfig{
+		SellBaseAssetCapInBaseUnits:  pointy.Float64(baseCap),
+		SellBaseAssetCapInQuoteUnits: pointy.Float64(quoteCap),
+		mode:                         volumeFilterModeExact,
+		additionalMarketIDs:          []string{},
+		optionalAccountIDs:           []string{},
 	}
+}
+
+func TestVolumeFilterFn(t *testing.T) {
+	dailyVolumeByDateQuery := mustMakeTestDailyVolumeQuery([]string{}, []string{})
```

Comment:
let's hold off on changes to TestVolumeFilterFn, may be better to move it to a separate PR since it's hard to follow and many of the comments that apply to the structure of the test above apply here. I think that what we learn from an improved structure to the first test will make writing and reviewing this test much easier -- this test is also a lot more complex because it's the main functionality and deals with numbers.

Reply:
I made some further changes that mostly simplified the interface but also rewrote `VolumeFilterFn` as a standalone function rather than a method. If you still want to hold off on changes, can do so, but might be good to just get it done in one swoop since it's massively simplified.